File: src/main/java/ru/ioleksiv/telegram/bot/api/model/objects/inline/queryresult/CaptionInlineResult.java
package ru.ioleksiv.telegram.bot.api.model.objects.inline.queryresult;

import com.fasterxml.jackson.annotation.JsonProperty;

public abstract class CaptionInlineResult extends ContentInlineResult {
    /**
     * caption	String	Optional. Caption, 0-200 characters
     */
    @JsonProperty("caption")
    private String caption = null;

    CaptionInlineResult(String type) {
        super(type);
    }

    public final String getCaption() {
        return caption;
    }

    public final void setCaption(String caption) {
        this.caption = caption;
    }
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/objects/inline/queryresult/CaptionInlineResult.kt
package io.github.oleksivio.telegram.bot.api.model.objects.inline.queryresult

import com.fasterxml.jackson.annotation.JsonProperty

abstract class CaptionInlineResult : ContentInlineResult() {
    /**
     * caption [String] Optional. Caption, 0-200 characters
     */
    @JsonProperty("caption")
    var caption: String? = null
}
